README.md

```diff
@@ -1,46 +1,118 @@
-keeps all user activity private.
+- **Mobile**: Users can connect to your mobile app through a deeplink to the 
+dapp browser in Coinbase Wallet Mobile App.
 
-For DApp developers to integrate with WalletLink, all you need to do is drop a
-few lines of code into your application, and WalletLink will take care of the
-rest. WalletLink is open-source and uses minimal dependencies for maximum
-security and no code bloat.
+- **Desktop**: Users can connect to your desktop app with a QR code in the 
+Coinbase Wallet Mobile App or with the Coinbase Wallet Chrome Extension.
 
-## Getting Started
+Wallet SDK is open-source and uses minimal dependencies for maximum security 
+and no code bloat. Simply drop a few lines of code into your dapp and Wallet 
+SDK takes care of the rest. 
 
-### Installation
+## Installing and Upgrading
+
+> Coinbase Wallet SDK was previously known as WalletLink. The installation 
+package is named `walletlink`. 
+
+### Installing Wallet SDK
+
+
+Install Coinbase Wallet SDK with yarn or npm. 
+
+#### Yarn
+
+1. Check available versions of Wallet SDK.
 
 ```shell
-# With Yarn
+yarn info walletlink versions
+```
+
+2. Install a specific version or the latest version.
+
+```shell
+#yarn add walletlink@2.4.6
 yarn add walletlink
+```
+
+3. Check your installed version. 
+
+```shell
+yarn list walletlink
+```
+
+#### Npm
+
+1. Check available versions of Wallet SDK.
+
+```shell
+npm view walletlink versions
+```
 
-# With NPM
+2. Install a specific version or the latest version.
+
+```shell
+#npm install walletlink@2.4.6
 npm install walletlink
 ```
 
-The following instructions are in [TypeScript](https://www.typescriptlang.org/),
-but the usage is the same in JavaScript, except for the occasional type
-annotations, for example `: string[]` or `as any`.
+3. Check your installed version. 
+
+```shell
+npm list walletlink
+```
+
+### Upgrading Wallet SDK
 
-### Upgrading from WalletLink 1.0 to 2.0
+Upgrade Coinbase Wallet SDK with yarn or npm. 
 
-For most users, simply update the NPM package, and you should be good to go.
+#### Yarn
 
-If you were using `ethereum.on("accountsChanged")`, please remove it and obtain
-addresses via [EIP-1102](#use-eip-1102-to-obtain-authorization-and-get-ethereum-accounts)
-callbacks instead. It's removed to improve compatibility with the latest web3.js.
+1. Compare your installed version of Coinbase Wallet SDK with the latest available version.
 
-Dark mode theme (`darkMode`) is now available as an option to WalletLink
-constructor.
+```shell
+yarn outdated walletlink
+```
+
+2. Update Coinbase Wallet SDK to the latest.
+
+```shell
+yarn upgrade walletlink --latest
+```
+
+#### Npm
+
+1. Compare your installed version of Coinbase Wallet SDK with the latest available version. 
+
+```shell
+npm outdated walletlink
+```
+
+2. If necessary, update `package.json` with the latest major version. 
+
+```shell
+{
+  "dependencies": {
+    "walletlink": "^2.0.0"
```

Comment:
should probably specify the most recent version here 2.4.7

Reply:
we should be able to add a step to `build:prod` script in package.json that automatically updates the version. Just replace each instance of `cur_version` with `next_version` (cc @bangtoven )

Reply:
@brendanww , sorry, are you saying I should use a variable? `cur_version`?

Reply:
Sorry, I am suggesting use the current latest version, and we'll make a script to update the README each time we publish a new version